support/Pipelines/Bbh/Inspiral.yaml

```diff
@@ -116,7 +116,7 @@ DomainCreator:
             - {{ ExcisionAShapeSpin_y }}
             - {{ ExcisionAShapeSpin_z }}
 {% endif %}
-        SizeInitialValues: [0.0, 0.0, 0.0]
+        SizeInitialValues: [0.0, -0.1, 0.0]
```

Comment:
@knelli2 I had to set this as well to get the control systems to start. Do you have a better idea how to do this; and do you know what SpEC does here? @markscheel do you know?

Reply:
Sometimes you have to do this in spec for high spins (make the excision surface move outward to avoid initial incoming char speeds), but I did not have to do this for the spec initial data I evolved

Reply:
I dropped this change and instead made the ID excisions bigger to match the value SpEC uses for negative-expansion boundary conditions. That seems to help.